src/helpers/daily-data-types.tsx

```diff
@@ -0,0 +1,81 @@
+	AppleHealthRemSleepMinutes = "AppleHealthSleepRemMinutes",
+	AppleHealthDeepSleepMinutes = "AppleHealthSleepDeepMinutes",
+	AppleHealthCoreSleepMinutes = "AppleHealthSleepCoreMinutes",
+	AppleHealthInBedMinutes = "AppleHealthInBedMinutes",
+	AppleHealthStandMinutes = "AppleHealthStandMinutes",
+	AppleHealthSteps = "AppleHealthSteps",
+	AppleHealthWalkingHeartRateAverage = "AppleHealthWalkingHeartRateAverage",
+	AppleHealthActiveEnergyBurned = "AppleHealthActiveEnergyBurned",
+	FitbitSedentaryMinutes = "FitbitSedentaryMinutes",
+	FitbitActiveMinutes = "FitbitActiveMinutes",
+	FitbitLightlyActiveMinutes = "FitbitLightlyActiveMinutes",
+	FitbitFairlyActiveMinutes = "FitbitFairlyActiveMinutes",
+	FitbitVeryActiveMinutes = "FitbitVeryActiveMinutes",
+	FitbitBreathingRate = "FitbitBreathingRate",
+	FitbitCaloriesBurned = "FitbitCaloriesBurned",
+	FitbitElevatedHeartRateMinutes = "FitbitElevatedHeartRateMinutes",
+	FitbitFatBurnHeartRateMinutes = "FitbitFatBurnHeartRateMinutes",
+	FitbitCardioHeartRateMinutes = "FitbitCardioHeartRateMinutes",
+	FitbitPeakHeartRateMinutes = "FitbitPeakHeartRateMinutes",
+	FitbitFloors = "FitbitFloors",
+	FitbitHrv = "FitbitHrv",
+	FitbitRestingHeartRate = "FitbitRestingHeartRate",
+	FitbitSleepMinutes = "FitbitSleepMinutes",
+	FitbitLightSleepMinutes = "FitbitLightSleepMinutes",
+	FitbitRemSleepMinutes = "FitbitRemSleepMinutes",
+	FitbitDeepSleepMinutes = "FitbitDeepSleepMinutes",
+	FitbitSpO2 = "FitbitSpO2",
+	FitbitSteps = "FitbitSteps",
+	GarminSteps = "GarminSteps",
+	GarminDistance = "GarminDistance",
+	GarminFloors = "GarminFloors",
+	GarminActiveMinutes = "GarminActiveMinutes",
+	GarminActiveCalories = "GarminActiveCalories",
+	GarminRestingCalories = "GarminRestingCalories",
+	GarminTotalCalories = "GarminTotalCalories",
+	GarminRestingHeartRate = "GarminRestingHeartRate",
+	GarminMinHeartRate = "GarminMinHeartRate",
+	GarminMaxHeartRate = "GarminMaxHeartRate",
+	GarminAverageHeartRate = "GarminAverageHeartRate",
+	GarminMaxStressLevel = "GarminMaxStressLevel",
+	GarminAverageStressLevel = "GarminAverageStressLevel",
+	GarminTotalStressMinutes = "GarminTotalStressMinutes",
+	GarminLowStressMinutes = "GarminLowStressMinutes",
+	GarminMediumStressMinutes = "GarminMediumStressMinutes",
+	GarminHighStressMinutes = "GarminHighStressMinutes",
+	GarminTotalSleepMinutes = "GarminTotalSleepMinutes",
+	GarminLightSleepMinutes = "GarminLightSleepMinutes",
+	GarminDeepSleepMinutes = "GarminDeepSleepMinutes",
+	GarminRemSleepMinutes = "GarminRemSleepMinutes",
+	GarminAwakeMinutes = "GarminAwakeMinutes",
+	GarminSleepScore = "GarminSleepScore",
+	GoogleFitSteps = "GoogleFitSteps",
+	Steps = "Steps",
+	RestingHeartRate = "RestingHeartRate",
+	SleepMinutes = "SleepMinutes",
+	HomeAirQuality = "HomeAirQuality",
+	WorkAirQuality = "WorkAirQuality"
+};
+
+export interface DailyDataTypeDefinition {
+	dataSource?: "Unified" | "AppleHealth" | "Garmin" | "Fitbit" | "GoogleFit" | "AirQuality";
```

Comment:
Sorry if I missed this in earlier PRs/discussions, but what is the "Unified" data source?

Reply:
Ah yeah I just updated this based on my work on the view builder PR.

I was trying to come up with a user-facing name for our "combined data sources".  Basically these query across Apple Health, Fitbit, Garmin to give you a single "Steps", "Resting Heart Rate", or "Sleep" value (these are the only "unified" data types at the moment).  Suspect we will add more in the future

"Unified" seems like a nice name for that I guess rather than "Combined" or "Deduplicated"

Reply:
Got it.  Make sense.